### PR TITLE
CASE is now Manifest

### PR DESCRIPTION
**Pull request by brunobuddy**
Replaces the old CASE software by Manifest as the project name and vision changed.

### CODE REVIEW

**Review by kokomo123 — APPROVED** [review]
It looks fine to me, pending other thoughts from collaborators. Thank you for your contribution!

**Review by nodiscc — CHANGES_REQUESTED** [review]
```
ERROR:awesome_lint.py: Manifest: source_code_url is undefined
```

**Comment by kokomo123** [comment]
I did not catch that, oops!

**Review by Ki-er — CHANGES_REQUESTED** [review]
Apparently this review didnt publish so here it is

File: software/manifest.yml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+name: Manifest
+website_url: https://manifest.build
+description: A complete backend that fits into 1 YAML file.
+licenses:
+  - MIT
+tags:
+  - Software Development - Low Code
```
[the remark this answers is not in the document]
Comment (nodiscc) [reply software/manifest.yml:7]: @brunobuddy ping

[the remark this answers is not in the document]
Comment (brunobuddy) [reply software/manifest.yml:7]: @nodiscc @Ki-er thank you, I commited the suggestion. However somhehow I still have a conflict on the `case.yml` file that I deleted.

File: software/manifest.yml [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+  - source_code_url: https://github.com/mnfst/manifest
```
Comment (nodiscc) [inline software/manifest.yml:8]:
```suggestion
source_code_url: https://github.com/mnfst/manifest
```

**Review by nodiscc — APPROVED** [review]

**Review by kokomo123 — APPROVED** [review]
LGTM, no problems here.

**Comment by nodiscc** [comment]
Merged in https://github.com/awesome-selfhosted/awesome-selfhosted-data/commit/a8587a7fde789ee42859e1eb67132886e09f60c4
I mistakenly pushed another repo's git history to @brunobuddy's fork. https://github.com/brunobuddy/awesome-selfhosted-data/ is now a mirror of Shaarli's source code :ok_hand: . I suggest you delete your fork.

**Pull request closed without merging**